blocks/hooks/default-autocompleters.js

```diff
@@ -11,14 +11,19 @@ import { addFilter } from '@wordpress/hooks';
 /**
  * Internal dependencies
  */
-import { userAutocompleter } from '../autocompleters';
+import { blockAutocompleter, userAutocompleter } from '../autocompleters';
+import { getDefaultBlockName } from '../api/registration';
 
 const defaultAutocompleters = [ userAutocompleter ];
 
-function setDefaultCompleters( completers ) {
+function setDefaultCompleters( completers, blockName ) {
 	if ( ! completers ) {
 		// Provide copies so filters may directly modify them.
 		completers = defaultAutocompleters.map( clone );
+		// Add blocks autocompleter for Paragraph block
+		if ( blockName === getDefaultBlockName() ) {
```

Comment:
I love how this change speaks directly to the default block, where we want the block autocompleter. Thanks, @gziolo.